### DISCUSSION TITLE
[GOG] DUNGEON KEEPER 2 - low performance

### DISCUSSION THREAD

**Discussion by tomasszz** [Heroic-Games-Launcher/HeroicGamesLauncher#2762] [Q&A]
Hi all, I'm launching DK2 from GOG and experiencing low performance in the game's main menu.
Game is unplayable.

I've tried different Wine/Proton versions, but no luck.

Does anyone have a similar issue?

**Comment by imLinguin** [comment]
That's probably due to usage of compatibility layer, it's best to manually configure custom script to run native version of DOSBox instead
